### PR TITLE
Update mopp talk info

### PR DESCRIPTION
**Pull request by mopp**
:pray:

### CODE REVIEW

File: _data/y2019/sessions.yaml [hunk 1]
```diff
@@ -3,32 +3,31 @@
   start_time: 13:00
   end_time: 13:20
   duration: 00:20
-  title: Let me talk about my vimrc to grow your vimrc
+  title: Your Vim Is Only for You
```
Comment (mopp) [inline _data/y2019/sessions.yaml]: This capitalization is right?
please teach me if you know.

Comment (mattn) [reply _data/y2019/sessions.yaml]: ping @ujihisa

Comment (ujihisa) [reply _data/y2019/sessions.yaml]: maybe "Your Vim is only for Vim"

Comment (ujihisa) [reply _data/y2019/sessions.yaml]:
```suggestion
  title: Your Vim is Only for You
```

File: _data/y2019/timetable.csv [hunk 2]
```diff
@@ -4,7 +4,7 @@ Start,End,Dur.,Content,Speaker,Language,Interp.,SessionID,Misc.,Video,Slides
 10:10,11:00,00:50,Keynote - Vim Renaissance,Prabir Shrestha,English,,,,,
 11:10,12:00,00:50,Keynote - We can have nice things,Justin M. Keyes,English,,,,,
 12:00,13:00,01:00,Lunch break,,,no,,,,
-13:00,13:20,00:20,Let me talk about my vimrc to grow your vimrc,mopp,English,,,,,
+13:00,13:20,00:20,Your Vim Is Only for You,mopp,English,,,,,
```
Comment (ujihisa) [inline _data/y2019/timetable.csv]:
```suggestion
13:00,13:20,00:20,Your Vim is Only for You,mopp,English,,,,,
```

File: _data/y2019/sessions.yaml (hunk 1 above)
Comment (mopp) [reply _data/y2019/sessions.yaml]: :special_thanks: :+1:

**Comment by mopp** [comment]
Thanks for your revising!
Let's merge it.

**Pull request merged**